### DISCUSSION TITLE
Dati personali (per gestione tramite utente Custode)

### DISCUSSION THREAD

**Discussion by maelaborazioni** [globaleaks/globaleaks-whistleblowing-software#3903] [Q&A] [answered]
Buonasera,
sto cercando di impostare il discorso della visibilità dei dati personali previa accettazione del Custode.
Il meccanismo mi è chiaro e lo trovo molto furbo (anche se ovviamente un po' macchinoso per chi deve gestirlo ma è normale).
Quello che mi sfugge è lato questionario.
Per ottenere i dati soggetti ad approvazione devo inserire da modello il gruppo di domande "Vuoi dirci chi sei?".

1. è questo l'unico modo per ottenere il comportamento della gestione del Custode?
2. c'è modo di modificare questo gruppo di domande?
3. c'è modo di avere un solo campo, magari nel mezzo del questionario, che deve essere soggetto a controllo del custode (es. Codice Fiscale)?

Grazie!

**Comment by evilaliv3** [comment, marked as answer]
Per usufruire della funzionalità dovete usare il nostro modello "Vuoi dirci chi sei?" che potete personalizzare a piacimento.
Per farlo dovete istanziare il modello e utilizzare la funzione "Sostituisci template" in cui inserire l'ID del vostro modello personalizzato

**Comment by maelaborazioni** [comment]
Capito. L'unica cosa che non si può modificare è il radio button sì/no, corretto? Posso cambiare la label ma non il controllo.
Non è possibile dunque partire con una dropdown al posto del radio button group?
Chiedo perchè in quel modo non ci sarebbe già il "sì" selezionato di default...
Grazie mille!

**Comment by evilaliv3** [comment]
Al momento no non è possibile.
Tenuto conto del valore del suggerimento andremo a variare l'implementazione in un prossimo rilascio.

**Comment by maelaborazioni** [comment]
Grazie mille, gentilissimo come sempre.
